plugins/database-migration-1.4.0/LICENSE.txt~d02bb5bc58476325dd3fe88139bb5bd8ca2d3bec

```diff
@@ -0,0 +1,201 @@
+                                 Apache License

```

Comment:
Should this file have been committed?  It looks like a backup or merge artifact.